### DISCUSSION TITLE
Pass Authentication Credentials to the APIReference Component

### DISCUSSION THREAD

**Discussion by marclave** [scalar/scalar#684] [Ideas]
We should be able to pre-populate the APIReference component with auth credentials, to improve the UX for developer docs ✨

**Comment by juliusjoosten** [comment]
@marclave Hi! If I understand this correctly, it would be super awesome to have. In order to explore our APIs, our prospects need to fill in a couple of entity ids in the requests and also set a bearer token, and doing this manually every time they want to play around is cumbersome. Is there maybe some progress in this area, or do you have some advise on how I could build something into my app that pre-fills requests and headers?

**Comment by hanspagel** [comment]
done like a year ago :))
